openapi/paths/objects@{object_id}.yaml

```diff
@@ -23,4 +23,47 @@ get:
       $ref: '../components/responses/500InternalServerError.yaml'
   tags:
     - Objects
-  x-swagger-router-controller: ga4gh.drs.server
\ No newline at end of file
+  x-swagger-router-controller: ga4gh.drs.server
+
+post:
+  summary: Get info about a DrsObject through POST'ing a Passport.
+  description: >-
+    Returns object metadata, and a list of access methods that can be used to fetch object bytes.
+
+    Method is a POST to accomodate a JWT GA4GH Passport sent in the formData in order to authorize access.
+  operationId: PostObject
+  security:
+    - PassportAuth: []
+  responses:
+    200:
+      $ref: '../components/responses/200OkAccess.yaml'
+    202:
+      $ref: '../components/responses/202Accepted.yaml'
+    400:
+      $ref: '../components/responses/400BadRequest.yaml'
+    401:
+      $ref: '../components/responses/401Unauthorized.yaml'
+    403:
+      $ref: '../components/responses/403Forbidden.yaml'
+    404:
+      $ref: '../components/responses/404NotFoundAccess.yaml'
+    500:
+      $ref: '../components/responses/500InternalServerError.yaml'
+  tags:
+    - Objects
+  x-swagger-router-controller: ga4gh.drs.server
+  parameters:
+    - $ref: '../components/parameters/ObjectId.yaml'
+    - $ref: '../components/parameters/Expand.yaml'
+  requestBody:
+    required: true
+    content:
+      multipart/form-data:
+        schema:
+          type: object
+          properties:
+            tokens[]:
```

Comment:
Is `tokens` the best attribute name to use here? Since we are trying to think long term interop with other standards, we don't want a collision if another API is using `tokens` to mean something else.

Would `passport_tokens`, `passportTokens` be more appropriate?

Reply:
I'm on the fence.  I think "yes" but then I wonder, what if we want to support alternative token JWT formats?  Beyond just passport 1.0.  Calling it tokens is more generic.  But if everyone wants "passports" then I'm fine with that.

Reply:
`tokens` renamed to `passports` based on discussion on Aug 23rd Cloud call